### PR TITLE
Resilience project

### PR DESCRIPTION
**Pull request by brownbaerchen**
Jupyter notebooks! Yay, how fun! It's not the best interface for the CI pipeline: You have to put your notebook path into the `convert_notebooks.py` script, but then it gets linted with flake8 and executed and put into the docs folder as rst, from where you can just `.. include::` it wherever you want! The downside is that I could not get this pytest notebook thing to work, but assertion errors are still raised during execution while converting. So you know when a test fails, but it's not as pretty.

### CODE REVIEW

**Review by pancetta — CHANGES_REQUESTED** [review]
So, what's the matter with pytest for notebooks? What problems occur?

File: docs/source/index.rst [hunk 1]
```diff
@@ -41,8 +41,8 @@ Projects
    projects/SDC_showdown.rst
    projects/AllenCahn_Bayreuth.rst
    projects/performance.rst
-   projects/error_estimation.rst
    projects/PinTSimE.rst
+   projects/Resilience/Resilience.rst
```
[the remark this answers is not in the document]
Comment (brownbaerchen) [reply docs/source/index.rst:45]: The executed notebooks go in that folder also

[the remark this answers is not in the document]
Comment (pancetta) [reply docs/source/index.rst:45]: I see. Well, as long as the paths are still working, so be it.

File: .github/workflows/ci_pipeline.yml [hunk 2]
```diff
@@ -52,6 +52,7 @@ jobs:
       - name: Install dependencies
         run: |
           mamba install --yes --override-channel -c conda-forge --file requirements.txt
+          pip install flake8_nb
```
[the remark this answers is not in the document]
Comment (brownbaerchen) [reply .github/workflows/ci_pipeline.yml:56]: I tried putting flake8_nb with an underscore in requirements and that didn't work, but with dash, this works!

**Comment by brownbaerchen** [comment]
> 

```pytest HotRod.ipynb
...
ERROR: not found: /Users/thomasbaumann/Documents/pySDC/pySDC/projects/Resilience/HotRod.ipynb```
Super weird, I don't really understand this

**Comment by pancetta** [comment]
If I see this correctly, conversion of the notebooks fails AND the CI does not pick this up. Both is.. not good.

**Comment by brownbaerchen** [comment]
> If I see this correctly, conversion of the notebooks fails AND the CI does not pick this up. Both is.. not good.

Yes, sorry, I didn't test enough! But the notebook conversion is not executed in the current pipeline, so that's actually fine. Coming back to my earlier question: How to test a different pipeline?

**Comment by pancetta** [comment]
Simply create another .yml file in `.github` and it will run whenever you tell it to run (via flags, see the other two for examples).

**Pull request closed without merging**